config/routes.rb

```diff
@@ -6,7 +6,7 @@
   post 'os_selected_discovered_hosts'           => 'hosts#os_selected'
   post 'medium_selected_discovered_hosts'       => 'hosts#medium_selected'
 
-  get 'discovered_hosts/help', :action => :welcome, :controller => 'discovered_hosts'
+  get 'discovered_hosts', to: 'foreman_discovery/react#index'
 
```

Comment:
@lzap can you please if there is no discovered host, the welcome page is displayed and when there is one discovered host we see the table in place?

Reply:
I checked and it fails :( @laviro @sharvit any idea how i can  just show the welcome page when there is no discovered host present. When a discovered host is present, we need to show the discovered host table.

Reply:
as discussed offline, lets let the rails controller decide on the view for this PR,
next step would be to let React decide what to show - empty state when there's no hosts / discovered hosts page.